masserstein/spectrum.py

```diff
@@ -534,22 +534,22 @@ def fuzzify_peaks(self, sd, step):
         Applies a gaussian filter to the peaks, effectively broadening them
```

Comment:
Ok, yeah. PEP 257 suggests splitting docstring. This is the only chance to correct this!

Reply:
Have you ever heard about PEP 257? :-D

Reply:
Grzesiu, nie marudź. Seriously though: the docs being non-PEP-257 compliant is a completely separate issue from this PR. If you feel like it, create an issue in the issue tracker, find an assignee to do it, PR welcome. Blocking a code-related patch over the fact that you don't like the style of all the docs in the repo is thw wrong way to go on about it, however.

Reply:
Sam marudzisz :-D
Every time I review the code I do it broadly. I read the changes and write what I would change to make it better/bugless. I do not block anybody. The author may accept suggestions, or dismiss them, it's up to @mciach.